packages/api/src/routes/medical/internal-cw.ts

```diff
@@ -0,0 +1,157 @@
+import NotFoundError from "@metriport/core/util/error/not-found";
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import httpStatus from "http-status";
+import {
+  verifyCxProviderAccess,
+  verifyCxItVendorAccess,
+} from "../../command/medical/facility/verify-access";
+import { isOboFacility } from "../../domain/medical/facility";
+import { OrganizationModel } from "../../models/medical/organization";
+import { FacilityModel } from "../../models/medical/facility";
+import {
+  getOrganizationOrFail,
+  getOrganizationByOidOrFail,
+} from "../../command/medical/organization/get-organization";
+import { getFaciltiyByOidOrFail } from "../../command/medical/facility/get-facility";
+import { createOrUpdateCWOrganization } from "../../external/commonwell/command/create-or-update-cw-organization";
+import {
+  CWOrganization,
+  get as getCWOrganization,
+  parseCWEntry,
+} from "../../external/commonwell/organization";
+import { cwOrgActiveSchema } from "../../external/commonwell/shared";
+import { requestLogger } from "../helpers/request-logger";
+import { asyncHandler, getFrom } from "../util";
+import { getUUIDFrom } from "../schemas/uuid";
+
+const router = Router();
+
+async function getParsedOrg(oid: string): Promise<CWOrganization> {
+  const resp = await getCWOrganization(oid);
+  if (!resp) throw new NotFoundError("Organization not found");
+  return parseCWEntry(resp);
+}
```

Comment:
Should we move this to the external folder?

Reply:
Done.